03-specifying-sample-designs.Rmd

```diff
@@ -353,7 +358,7 @@ Consider a population where there are $N$ clusters and $n$ clusters are sampled
 $$\bar{y}=\frac{\sum_{i=1}^n M_i \bar{y}_i}{ \sum_{i=1}^n M_i}$$
 Note this is a consistent but biased estimator. Often the population size is not known, so this is a method to estimate a mean without knowing the population size. The estimated standard error of the mean is:
 
-$$se(\bar{y})=\frac{1}{\hat{N}_{pop} } \sqrt{\frac{N^2 (1-\frac{n}{N})}{n}\frac{1}{n-1} \sum_{i=1}^n (M_i\bar{y}_i -\hat{t}/N)^2 + \frac{N}{n} \sum_{i=1}^n \frac{M_i^2}{m_i}\left(1-\frac{m_i}{M_i}\right)s^2_i }$$
```

Comment:
Should we change the number of clusters to be something other than "N".  Currently confusing with the population size.

Reply:
@rpowell22 @szimmer would it make sense to use K/k for clusters?

Reply:
Switched to A/Bs